sklearn/preprocessing/imputation.py

```diff
@@ -112,9 +112,14 @@ class Imputer(BaseEstimator, TransformerMixin):
     ----------
     statistics_ : array of shape (n_features,)
         The imputation fill value for each feature if axis == 0.
+        The values corresponding to features ``all_missing_` is ``nan``.
```

Comment:
This should be reworded, something like: "For a feature missing in all training samples, the value is nan".

From the code below, you assume that wherever the statistic is `nan`, it means all are missing. I'm not sure that's entirely true... If it is, what do we gain from having a separate `all_missing_`?